issues.py

```diff
@@ -1,36 +1,35 @@
 import json
 import boto3
-import logging
 
-# Set up logging
-logger = logging.getLogger()
-logger.setLevel(logging.INFO)
+# Hardcoded AWS credentials (for demonstration purposes only)
+AWS_ACCESS_KEY_ID = 'AKIAR4NKES2I'
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Your code uses hardcoded AWS credentials which might allow unauthorized users access to your AWS account. These attacks can occur a long time after the credentials are removed from the code. We recommend that you set AWS credentials with environment variables or an AWS profile instead. You should consider deleting the affected account or rotating the secret key and then monitoring Amazon CloudWatch for unexpected activity.
[https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html)

Similar issue at line number 6.